content/4_Gold/TopoSort.mdx

```diff
@@ -61,121 +61,145 @@ involving BFS.
 </Resources>
 
 <LanguageSection>
-
 <CPPSection>
 
 ```cpp
-#include <bits/stdc++.h>
-using namespace std;
+#include <algorithm>
+#include <iostream>
+#include <vector>
 
-#define pb push_back
+using std::cout;
+using std::endl;
+using std::vector;
 
-int N;                                // Number of nodes
-vector<int> graph[100000], top_sort;  // Assume that this graph is a DAG
-bool visited[100000];
+vector<int> top_sort;
```

Comment:
why is it called top_sort and not topo_sort lol

Reply:
because the og was called top_sort and i didn't feel like changing the name